interfaces/Portal/src/app/app-routing.module.ts

```diff
@@ -122,7 +114,13 @@ const routes: Routes = [
         // Fallback for change in url, from old to new syntax:
         path: 'registration-validation',
```

Comment:
```suggestion
        path: 'registrationValidation',
```

This would be the new "old" URL. :)
The move from "registration-validation" to "registrationValidation" is a _very_ long time ago now... So that can go, I think.